lib/ActivityStream.jsm

```diff
@@ -251,6 +251,10 @@ const PREFS_CONFIG = new Map([
     skipBroadcast: true,
     value: "{}",
   }],
+  ["discoverystream.endpointSpocsClear", {
+    title: "Endpoint for when a user opts-out of sponsored content to delete the user's data from the ad server.",
+    value: "https://spocs-dev.getpocket.dev/user",
```

Comment:
Any reason for a separate pref (vs as part of `.config`? Or a pref that's specific to spocs clear vs generally some server endpoint that we append `/user`?

Reply:
I kept it separate to try to reduce the noise in the main `.config` There is already a lot in `.config` and I felt like this isn't something we would be changing based on a experiment.

I think moving it into the config would work, it would just make that config even bigger, which I thought would be nice to try to avoid.